### PR TITLE
October a11y bug fixes

### PR DESCRIPTION
**Pull request by cRui861**
October accessibility bug fixes including fixes for focus order, focus visual indicator, role updates for screen reader, tooltip overlap on hover, and placeholder suggestions for editable fields.

### CODE REVIEW

**Comment by dpwatrous** [comment]
Looks good to me! Just a couple of minor suggestions as far as the actual Git commits are concerned:

- Let's rebase this on master so we can use the 'create a merge commit' merge option and preserve individual commits for different a11y fixes (if we don't rebase we'll have 2 merge commits, which is going to look pretty confusing in Git history)

- Since the DevOps work items aren't visible from GitHub, let's reword the commit messages with a brief description of each fix. You can still link the DevOps work item in the commit's description field. See my commit in this PR for an example.

**Review by dpwatrous — CHANGES_REQUESTED** [review]
(See my comment above)

**Comment by cRui861** [comment]
> Looks good to me! Just a couple of minor suggestions as far as the actual Git commits are concerned:
> 
> * Let's rebase this on master so we can use the 'create a merge commit' merge option and preserve individual commits for different a11y fixes. If we don't rebase we'll have 2 merge commits, which is going to look pretty confusing in Git history.
> * Since the DevOps work items aren't visible from GitHub, let's reword the commit messages with a brief description of each fix. You can still link the DevOps work item in the commit's description field. See my commit in this PR for an example.

Sounds good. I'll make the updates to the commit messages.

**Review by gingi — APPROVED** [review]
Looks good. Just 3 suggestions for localization.

File: src/@batch-flask/ui/buttons/refresh-btn/refresh-btn.component.ts [hunk 1]
```diff
@@ -35,6 +35,8 @@ export class RefreshButtonComponent implements OnDestroy {
 
     @Input() public type: string = "square";
 
+    @Input() public title: string = "Refresh";
```
[the remark this answers is not in the document]
Comment (cRui861) [reply src/@batch-flask/ui/buttons/refresh-btn/refresh-btn.component.ts:38]: Could you point me to an example of what you mean by this?

File: src/app/components/account/details/account-summary-card/account-summary-card.html [hunk 2]
```diff
@@ -15,7 +15,7 @@
         </div>
     </div>
     <bl-button-group summaryActions>
-        <bl-refresh-btn [refresh]="refresh"></bl-refresh-btn>
+        <bl-refresh-btn [refresh]="refresh" title="Refresh account summary page"></bl-refresh-btn>
```
[the remark this answers is not in the document]
Comment (cRui861) [reply src/app/components/account/details/account-summary-card/account-summary-card.html]: Added localization for this refresh button.

File: src/app/components/account/details/programatic-usage/programing-sample/programing-sample.html [hunk 3]
```diff
@@ -5,7 +5,7 @@
     <bl-property-group label="" class="wide" class="prerequisites">
         <bl-table-property label="Prerequisites" *ngIf="prerequisites">
             <bl-table-property-row *ngFor="let prerequisite of prerequisites;trackBy: trackPrerequisite">
-                <bl-tp-cell [value]="prerequisite"></bl-tp-cell>
+                <bl-tp-cell [value]="prerequisite" role="button" attr.aria-label="Copy {{prerequisites}} to clipboard"></bl-tp-cell>
```
[the remark this answers is not in the document]
Comment (cRui861) [reply src/app/components/account/details/programatic-usage/programing-sample/programing-sample.html]: Added localization for this arial label.

**Review by zfengms — APPROVED** [review]

**Comment by cRui861** [comment]
Fixes #2415

**Pull request merged**